### ISSUE TITLE
phpunit configuration file (minor)

### ISSUE THREAD

**Issue by remicollet** [doctrine/DoctrineCacheBundle#150]
Minor issue with the phpunit.xml

```
PHPUnit 7.4.3 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.3.0RC5
Configuration: /dev/shm/BUILD/DoctrineCacheBundle-5514c90d9fb595e1095e6d66ebb98ce9ef049927/phpunit.xml.dist

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 11:
  - Element 'phpunit', attribute 'syntaxCheck': The attribute 'syntaxCheck' is not allowed.

  Test results may not be as expected.


...................................................SSSS...SS..    62 / 62 (100%)

Time: 193 ms, Memory: 18.00MB

There were 6 skipped tests:
...

```

**Comment by alcaeus** [comment]
This will be fixed in #147, but only for master. Would it make sense to apply the fix to 1.3 as well?

**Comment by remicollet** [comment]
> Would it make sense to apply the fix to 1.3 as well?

@alcaeus IMHO yes (no risk)

**Comment by alcaeus** [comment]
Fixed in #151.
